exercise-1.js

```diff
@@ -293,27 +434,38 @@
         ['android'] => chessCalc() => null
 
         */
+        
 
-        function chessCalc(pieces) {
-            //your code here
-        }
-
+        // function chessCalc(pieces) {
+        // };
 
+        // chessCalc();
 
 
         /************************************************************* */
 
         const ones = [1,11,111,1111,11111,111111,1111111,11111111,111111111,1111111111];
         //reverse the array, without modifying / `mutating` the ones array.
-
-
+        function reverseArrayFunction(array){
+            let reverseArray = [];
+            for(let i = array.length-1; i >= 0; i--){
+                reverseArray.push(array[i]);
+            } return reverseArray
+        }
+        
+        console.log(reverseArrayFunction(ones));
 
         /************************************************************* */
         //create a function called performer(cb) that takes in a callback function and runs that callback function.  The function should return the output of the callback function.
 
         function performer(cb) {
-            //code goes here
+            let name = this.name;
+            return name;
+        };
+        function callback(name){
+            console.log(`Hi my name is ${name}`);
         }
+        performer(callback('Jorge'));
```

Comment:
The expectation here is for performer to simply take in a 'function' as a parameter, execute the function and return the results of that function.